torch/csrc/inductor/aoti_runtime/interface.h

```diff
@@ -52,6 +54,15 @@ AOTIRuntimeError AOTInductorModelContainerCreate(
     bool is_cpu,
     const char* cubin_dir);
 
+// Creates an AOTInductor model container. The parameter num_models
+// specifies the number of model instances that may be run concurrently for
+// the same input model.
+AOTIRuntimeError AOTInductorModelContainerCreateWithDevice(
+    AOTInductorModelContainerHandle* container_handle,
+    size_t num_models,
+    const char* device_str,
```

Comment:
What happens if `device_str == nullptr`? Let's document the behavior by adding comments.

Reply:
commented. 
device_str cannot be nullptr, and it must be a valid device str.